seedvault/src/main/java/com/solanamobile/seedvault/Bip32DerivationPath.java

```diff
@@ -123,7 +123,7 @@ public static Bip32DerivationPath fromUri(@NonNull Uri bip32Uri) {
             throw new UnsupportedOperationException("BIP32 URI must be hierarchical");
         }
 
-        if (bip32Uri.isAbsolute() && !bip32Uri.getScheme().equals(WalletContractV1.BIP32_URI_SCHEME)) {
```

Comment:
Can results in a NPE if `getScheme` returns null. Switching the order guarantees that `equals` is never called on a null object reference.